geospace-http/src/main/scala/com/socrata/geospace/http/client/CoreServerClient.scala

```diff
@@ -117,7 +117,7 @@ class CoreServerClient(httpClient: HttpClient,
       case Some(rb) =>
         val request = buildRequest(rb)
         logger.info("Request: " + request)
-        for (response <- httpClient.execute(request)) yield {
+        for (response <- httpClient.execute(request)) {
```

Comment:
That's actually the big change in simple-arm v2; `Managed#map` changed from having type `[B](f: A => B): B` to `[B](t: A => B): Managed[B]` (i.e., it's a functorial map, not a way to run the compuation).  Which meant that a new way to run the compuation needed to be introduced; `Managed#run` is the canonical way to do that, but since `(for(...) yield { blah }).run` is hideous and `foreach` is not fundamental to a monadic compuation in the way `map` and `flatMap` are (indeed for most monads `foreach` makes essentially no sense), I made `Managed#foreach` return the value.